### PR TITLE
Backend Update

### PR DESCRIPTION
**Pull request by LostMailman**
1. API w/ sample Json
2. Parsing mta data - regex function. Still working on cleaning and turning into data

### CODE REVIEW

File: server/test.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#html('table')
+#dfHTML = pd.read_html(html)
+#lsTables = [pd.read_html(str(html('table')[x])) for x in 
+
+lsHTML = [BeautifulSoup(text, 'html.parser') if bool(text) else None for text in lsText]
+
+dfFeed = pd.DataFrame(zip(lsName, lsStatus, lsHTML, lsDate, lsTime), columns = ['Name', 'Status', 'HTML', 'Date', 'Time'])
+
+#Consider only MTA subway lines
+dfFeed = dfFeed.truncate(after=9)
```
Comment (kz363) [inline server/test.py]: I believe this cuts off the S train status. Should be 10?

**Comment by zmln** [comment]
imo, test code like in `server/app.py` shouldn't be added

**Comment by kz363** [comment]
imo tests are pretty useless for Charles' parsing code since he has to stub data (the serviceStatus.txt refreshes every minute). No point in writing tests that parse the same stubbed data every time.

The tests that matter later are the integration tests that will generate the json for the map.

**Comment by zmln** [comment]
I think tests with fake data are still useful. It checks that the functions is returning consistent data even when we modify the function. Let's say we always want to make sure time is present in the returned data.

```
def parse(data):
  ...

parsed_data = parse(stubbedData)
expect(parsed_data['time']).is.not.None
etc.
```

When we modify parse and run these tests again, it test whether or not we messed up with the time field.

File: server/test.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import urllib
+from bs4 import BeautifulSoup
+import pandas as pd
+
+response = urllib.urlopen('http://web.mta.info/status/serviceStatus.txt')
```
Comment (zmln) [inline server/test.py]: Most of this module should be in a function that is exported.

if you import this module, this line will only run once and then cached the next time you import it. This basically means you are requesting this url once and not getting updates.

**Pull request merged**